tests/driver/api.test.ts

```diff
@@ -679,7 +679,7 @@ describe('Mongoose Model API level tests', async () => {
     describe('vector search', function() {
         const vectorSchema = new Schema(
             {
-                $vector: { type: [Number], default: () => void 0 },
+                $vector: { type: [Number], default: () => void 0, select: true },
```

Comment:
I understand, we don't have a way to test $vectorize at this point. Is there a way to add a unit test to the plugin function to verify if both fields are added as expected?

Reply:
Not that I know of unfortunately.

Reply:
Sure, probably we can add tests around $vectorize later. Can we create an issue for $vectorize testing in general and strategize how we are going to do that from stargate-mongoose? Currently, $vectorize is not covered in any of our tests if I'm not wrong.

Reply:
Yeah $vectorize isn't. I think you mentioned there's an Astra CLI that would be helpful for testing $vectorize, do you have any docs for that?

Reply:
I don't have it right now, we need to collect some details about using Astra DevOps API, having a token securely, etc. We can work on the details and for now I think we can just create a ticket and move on.